apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/LoggingSnippets.cs

```diff
@@ -123,58 +131,21 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
     }
 
     /// <summary>
-    /// A simple web application to test the <see cref="GoogleLogger"/>
-    /// and associated classes in ASP.Net Core 2.
-    /// </summary>
-    internal class LoggingTestApplicationCore2
-    {
-        private static readonly string ProjectId = Utils.GetProjectIdFromEnvironment();
-
-        // Sample: RegisterGoogleLogger2
-        public void ConfigureServices(IServiceCollection services)
-        {
-            // Replace ProjectId with your Google Cloud Project ID.
-            services.AddSingleton<ILoggerProvider>(sp => GoogleLoggerProvider.Create(sp, ProjectId));
-
-            services.AddMvc();
-        }
-
-        public void Configure(IApplicationBuilder app)
-        {
-            // ...any configuration your application requires.
-
-            app.UseMvc(routes =>
-            {
-                routes.MapRoute(
-                    name: "default",
-                    template: "{controller=Home}/{action=Index}/{id?}");
-            });
-        }
-        // End sample
-    }
-
-    /// <summary>
-    /// A controller for the <see cref="LoggingTestApplication"/>
-    /// or for the <see cref="LoggingTestApplicationCore2"> used to log information.
+    /// A controller for the <see cref="LoggingTestApplication"/> used to log information.
     /// </summary>
+    // Sample: UseGoogleLogger
     public class LoggingSamplesController : Controller
     {
-        internal const string LoggerName = "Logger for Testing";
-
-        // Sample: UseGoogleLogger
         /// <summary>
         /// This method logs an Information level message.
-        /// The <see cref="ILoggerFactory"/> is populated by dependency injection
+        /// The <see cref="ILogger"/> is populated by dependency injection
         /// thanks to the use of the <see cref="FromServicesAttribute"/> attribute.
         /// </summary>
-        public void LogInformation(string id, [FromServices]ILoggerFactory loggerFactory)
+        public void LogInformation(string id, [FromServices]ILogger<LoggingSamplesController> logger)
```

Comment:
I *think you can just do `ILogger logger`.  I think this is what most users will do